src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1011"
+  },
+  "description": {
+    "text": "An empty project template for creating an ASP.NET Core application. This template does not have any content in it.",
```

Comment:
Maybe we should add F# somewhere in the description cc @Eilon

Reply:
Spoke offline. Won't be required.

Reply:
I am curious how this will be localized.  Is this taken care of or is there more work that needs to be done?

Reply:
@mlorbetske / @seancpeters would likely know the answer to that.

Reply:
This is localized through the VS package which is carried in the Web workload. As long as we don't change the names or descriptions from their C# versions, we can just use the same package ids and resource ids here. Note that the text property in the json is only used if the package/resource id combination cannot be resolved by VS.